dataland-frontend/tests/e2e/utils/EuTaxonomyFinancialsUpload.ts

```diff
@@ -281,9 +281,27 @@ export function uploadCompanyViaApiAndEuTaxonomyDataForFinancialsViaForm(
 /**
  *
  * @param data the data to fill the form with
+ * @param reportingPeriod (optional) to specify reporting period
  */
-export function fillEuTaxonomyForFinancialsRequiredFields(data: EuTaxonomyDataForFinancials): void {
+export function fillEuTaxonomyForFinancialsRequiredFields(
```

Comment:
I think this function is unnecessary after you switch to Api-Uploads in the test instead of filling the whole EuTaxoFinancials-form

Reply:
Done.

Reply:
Reminder for myself to check:  Check if extension and adjustments and renaming to "fillAndValidateEuTaxonomyCreditInstitutionForm()" still necessary after last commits